RefactorTest/RenameCpp20.cpp

```diff
@@ -278,7 +278,7 @@ void f8()
     {
         // #TEST#: R937 Rename local variable vals
         // #TEST#: R938 Rename use of one
-        const int vals[3](one, 2, 3);
+        const int vals[3]{one, 2, 3};
```

Comment:
Don't change these.  The comment on line 273 explicitly says that this is initialization via parentheses.